docs/pages/tutorial/create-a-modal.mdx

```diff
@@ -332,11 +332,10 @@ The modal allows the user to choose an emoji from a list of available emoji. Cre
 import { Modal, View, Text, Pressable, StyleSheet } from 'react-native';
 import MaterialIcons from '@expo/vector-icons/MaterialIcons';
 
-type Props = {
+type Props = PropsWithChildren<{
```

Comment:
Thanks for the update! 

We also need to add `import` from 'react' with this notation, alternatively you can use `React.PropsWithChildren`.

Reply:
Thank you so much for your invaluable support, @Simek. I truly appreciate your guidance and assistance throughout this process. I have used the `import` from 'react' for consistence.